frontends/web/src/routes/account/info/info.jsx

```diff
@@ -37,12 +38,8 @@ function SigningConfiguration({ t, signingConfiguration }) {
                                 {t('accountInfo.extendedPublicKey')}
                                 {signingConfiguration.xpubs.length > 1 && (' #' + (index+1))}
                             </strong><br />
-                            <QRCode data={xpub} /><br />
-                            <Input
-                                readOnly
-                                className={style.addressField}
-                                onFocus={focus}
-                                value={xpub} />
+                            <QRCode data={xpub} />
+                            <CopyableInput value={xpub} />
```

Comment:
Didn't you lose `className={style.addressField}` here? Please ignore if this was intentional (because you styled the elements from a different place).